packages/docs/static/data/resourceDefinitions/coverage.json

```diff
@@ -1,12 +1,15 @@
 {
-  "resourceName": "Coverage",
+  "name": "Coverage",
+  "location": "resource",
   "description": "Financial instrument which may be used to reimburse or pay for health care products and services. Includes both insurance and self-payment.",
   "properties": [
     {
       "name": "Coverage",
       "depth": 0,
       "types": [
-        ""
```

Comment:
@rahul1 How would you feel about just using the original `StructureDefinition` and `ElementDefinition` rather than converting to this custom type?  It looks basically the same?

Reply:
Yeah, originally this structure was a lot smaller and simpler. I was also afraid of the performance impacts of having the 

I'd be in favor of using the StructureDef directly, but I think we should still split them up into indvidual json files, rather than having each page load the whole "profiles-*" file.

I'll make this change in a future PR